pkg/cmd/admin/router/router.go

```diff
@@ -438,7 +440,8 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 			return fmt.Errorf("router could not be created; you must specify a service account with --service-account")
```

Comment:
It will also need to use the host ports when using --host-network=false. Which will also need the --service-account option.